srcpkgs/oil/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'oil'
 pkgname=oil
-version=0.8.6
+version=0.8.7
 revision=1
 build_style=configure
-configure_args="--prefix=/usr $(vopt_with readline)"
-hostmakedepends="$(vopt_if readline readline-devel) python"
+configure_args="--prefix=/usr"
```

Comment:
Why did you remove readline?

Also, the commit message should note that these changes are being made.

Reply:
My bad, I thought I did that one only on personal branch for testing.